package-lock.json

```diff
@@ -6929,7 +6929,8 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true
+          "bundled": true,
```

Comment:
Package locks without package changes should not be committed. Please remove this change.

Reply:
done